README.md

```diff
@@ -124,9 +124,10 @@ All configuration values are optional, though an [auth key] is strongly recommen
 If no auth key is present, one will be loaded from the default `$TS_AUTHKEY` environment variable.
 Failing that, it will log an auth URL to the Caddy log that can be used to register the node.
 
-After the node had been added to your network, you can restart Caddy without the debug logging.
 Unless the node is registered as `ephemeral`, the auth key is only needed on first run.
 Node state is stored in `state_dir` and reused when Caddy restarts.
+When running in a container, it is generally recommended to use `ephemeral` and always provide an auth key,
+or to mount the state directory on a persistent volume, depending on the use case.
```

Comment:
(Note to self): We should probably document these practices (and the why)  somewhere in the main docs page so that places like this can just link it. I've created an issue for this https://github.com/tailscale/tailscale-www/issues/6107 (closed source)